CodeSniffer/Standards/Squiz/Sniffs/Commenting/FunctionCommentSniff.php

```diff
@@ -238,6 +246,22 @@ protected function processThrows(PHP_CodeSniffer_File $phpcsFile, $stackPtr, $co
      */
     protected function processParams(PHP_CodeSniffer_File $phpcsFile, $stackPtr, $commentStart)
     {
+        if ($this->_platform === null) {
```

Comment:
other sniffs do this at the top of `process()` but this sniff does not have such a method. Do you still recommend I move it into the constructor, or should I somehow add a `process()` method to this sniff?